recipes/python-lzo/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "python-lzo" %}
+{% set version = "1.11" %}
+{% set sha256 = "38a0ea4ceb27cdd8e3526509fe1b7a936e5dfa57c64608fd32085c129e8be386" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - lzo 2.10.*
+  run:
+    - python
+    - lzo 2.10.*
```

Comment:
I think we are using `2.*` instead of `2.10.*`. The latter is more restrictive and it is fine if your package needs that.